qiskit/providers/models/backendproperties.py

```diff
@@ -182,6 +183,8 @@ def __init__(self, backend_name, backend_version, last_update_date, qubits,
         """
         self.backend_name = backend_name
         self.backend_version = backend_version
+        if isinstance(last_update_date, str):
+            last_update_date = dateutil.parser.parse(last_update_date)
```

Comment:
So I did a little digging into marshmallow and it only would except iso8601 string: https://marshmallow.readthedocs.io/en/stable/api_reference.html#marshmallow.fields.DateTime (based on our usage with the default: https://github.com/Qiskit/qiskit-terra/blob/77f8c0875ceabe9f14ff4b49fef5352b53dfd897/qiskit/validation/fields/__init__.py#L60 ). I'm thinking we might want to be careful here and not accept arbitrary input strings and expand what we have to support moving forward. Can we change this to `isoparse` https://dateutil.readthedocs.io/en/stable/parser.html#dateutil.parser.isoparse ?

Reply:
I thought I was using `isoparse`... fixed now.